doc/tagbar.txt

```diff
@@ -505,20 +507,20 @@ Example:
 g:tagbar_height~
 Default: 0
 
-If |g:tagbar_position| is set to 'bottom' or 'top', then this value is used to
-determine the height of the Tagbar window.
-See |g:tagbar_left| for configuring the position of the window.
+If |g:tagbar_position| is does not include a 'vertical' option, then this
```

Comment:
very minor typo here...
```
If |g:tagbar_position| does not include a 'vertical' option, then this
```
(remove is)

Reply:
Please apply this suggestion and I'll merge --- the rest looks good to me.

Reply:
fixed